### PR TITLE
added axiom for ring chromosomes

### PR DESCRIPTION
**Pull request by sabrinatoro**
addresses #3507

Still need to do:
'ring chromosome' should not be in the "autosomal" branch.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -440960,6 +441008,19 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/65
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2614 xsd:string
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700091
+name: disorder caused by ring chromosome
+def: "Chromosomal anomaly consisting of the presence of a ring chromosome. A ring chromosome is a chromosome whose arms have fused together to form a ring, often with the loss of the ends of the chromosome." [https://orcid.org/0000-0002-4142-7153, NCIT:C3360]
+synonym: "r" EXACT [NCIT:C3360]
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:441015]: definitely not EXACT. I am not sure whether this buys us anything at all

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -440960,6 +441008,19 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/65
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2614 xsd:string
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700091
+name: disorder caused by ring chromosome
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:441013]: non-blocking: would prefer a less awkward primary label like "ring chromosome anomaly:

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -440960,6 +441008,19 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/65
...
+def: "Chromosomal anomaly consisting of the presence of a ring chromosome. A ring chromosome is a chromosome whose arms have fused together to form a ring, often with the loss of the ends of the chromosome." [https://orcid.org/0000-0002-4142-7153, NCIT:C3360]
+synonym: "r" EXACT [NCIT:C3360]
+synonym: "supernumerary circular chromosome" EXACT [NCIT:C3360]
+xref: MESH:D012303 {source="MONDO:equivalentTo"}
+xref: NCIT:C3360 {source="MONDO:equivalentTo"}
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:441018]: is this exact? is this in the disorder branch?

I think it's OK to go in as exact for now but we need to return to the issues of mappings to other branches and conflation

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -440960,6 +441008,19 @@ relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/65
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/2614 xsd:string
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700091
+name: disorder caused by ring chromosome
+def: "Chromosomal anomaly consisting of the presence of a ring chromosome. A ring chromosome is a chromosome whose arms have fused together to form a ring, often with the loss of the ends of the chromosome." [https://orcid.org/0000-0002-4142-7153, NCIT:C3360]
+synonym: "r" EXACT [NCIT:C3360]
+synonym: "supernumerary circular chromosome" EXACT [NCIT:C3360]
+xref: MESH:D012303 {source="MONDO:equivalentTo"}
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:441017]: also UMLS:C0035639 ?

**Review by cmungall — APPROVED** [review]
This is good to go other than the one synonym issue that can be fixed later

I assume DPs coming later

I think we also need to be more explicit about germline vs somatic

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:441013]: label updated

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:441018]: There are a lot of NCIT and MESH ids that have been used as "disease or disorder" but are actually not a disorder. This is especially an issue in the chromosomal anomaly branch. I have made a note that we should come back to this specific issue.

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:441017]: added

**Comment by nicolevasilevsky** [comment]
@sabrinatoro is this ready to merge?

**Pull request merged**